### PR TITLE
INGK-517 Allow multiple servos ethernet network

### PR DESCRIPTION
**Pull request by MartinALL91**

### CODE REVIEW

File: ingenialink/ethernet/network.py [hunk 1]
```diff
@@ -41,32 +43,36 @@ class NetStatusListener(Thread):
...
+                    if ping_response:
+                        time.sleep(0.25)
+                    else:
+                        servo_state = NET_STATE.DISCONNECTED
+                        self.__network._notify_status(servo_ip,
+                                                      NET_DEV_EVT.REMOVED)
+                elif ping_response:
+                    servo_state = NET_STATE.CONNECTED
+                    self.__network._notify_status(servo_ip,
+                                                  NET_DEV_EVT.ADDED)
```
Comment (GullinBustin) [inline ingenialink/ethernet/network.py:73]: I think you can simplify the if:
```python
if servo_state == NET_STATE.CONNECTED and not ping_response:
    self.__network._notify_status(servo_ip, NET_DEV_EVT.REMOVED)
    self.__network._set_servo_state(servo_ip, NET_STATE.DISCONNECTED)
if servo_state == NET_STATE.DISCONNECTED and ping_response:
    self.__network._notify_status(servo_ip, NET_DEV_EVT.ADDED)
    self.__network._set_servo_state(servo_ip, NET_STATE.CONNECTED)


 ```

**Review by GullinBustin — CHANGES_REQUESTED** [review]

File: ingenialink/ethernet/network.py (hunk 1 above)
Comment (MartinALL91) [reply ingenialink/ethernet/network.py:73]: This does not contemplate the sleep function.

**Review by WoodElv — APPROVED** [review]

File: ingenialink/ethernet/network.py [hunk 2]
```diff
@@ -41,32 +43,36 @@ class NetStatusListener(Thread):
...
+                    response = ping(servo_ip, timeout=1)
+                    if not isinstance(response, float):
+                        unsuccessful_pings += 1
+                    else:
+                        break
+                ping_response = (unsuccessful_pings !=
+                                 self.__max_unsuccessful_pings)
+                if servo_state == NET_STATE.CONNECTED:
+                    if ping_response:
+                        time.sleep(0.25)
```
[the remark this answers is not in the document]
Comment (GullinBustin) [reply ingenialink/ethernet/network.py:65]: I understand that you don't want to do the ping without sleep, but, why don't do sleep always?

File: ingenialink/ethernet/network.py (hunk 1 above)
Comment (MartinALL91) [reply ingenialink/ethernet/network.py]: Done

**Review by GullinBustin — APPROVED** [review]

**Pull request merged**